### PR TITLE
Add custom goverance to action configs

### PR DESCRIPTION
**Pull request by usefulalgorithm**
### 🤔 Why?

Add custom governance related configurations.

### 🤓 What?

As titled.

### 🧪 Tested?

N/A.

### CODE REVIEW

**Review by mars-lan — COMMENTED** [review]
Also need to add it to the actual workflow: https://github.com/MetaphorData/connectors-action/blob/main/.github/workflows/shared.yml

**Comment by mars-lan** [comment]
Please run https://github.com/MetaphorData/connectors-action/actions/workflows/manual.yml once to verify that it's working end-to-end.

**Comment by usefulalgorithm** [comment]
@mars-lan  it works: https://github.com/MetaphorData/connectors-action/actions/runs/6853041811

**Comment by mars-lan** [comment]
> @mars-lan it works: https://github.com/MetaphorData/connectors-action/actions/runs/6853041811

Didn't seem to populate? https://stage.metaphor.io/dataset/8CDF81FE6B8050607E78C53E8B704B05/columns

**Comment by usefulalgorithm** [comment]
> > @mars-lan it works: https://github.com/MetaphorData/connectors-action/actions/runs/6853041811
> 
> Didn't seem to populate? https://stage.metaphor.io/dataset/8CDF81FE6B8050607E78C53E8B704B05/columns

I ran it against QA: https://qa.metaphor.io/dataset/8CDF81FE6B8050607E78C53E8B704B05/columns

**Comment by mars-lan** [comment]
> > > @mars-lan it works: https://github.com/MetaphorData/connectors-action/actions/runs/6853041811
> > 
> > 
> > Didn't seem to populate? https://stage.metaphor.io/dataset/8CDF81FE6B8050607E78C53E8B704B05/columns
> 
> I ran it against QA: https://qa.metaphor.io/dataset/8CDF81FE6B8050607E78C53E8B704B05/columns

I see. Seems like the tags are not populate, probably because the corresponding tags don't exist yet. Consider verify using an existing tag, like `Classification=Public`.

**Comment by usefulalgorithm** [comment]
> > > > @mars-lan it works: https://github.com/MetaphorData/connectors-action/actions/runs/6853041811
> > > 
> > > 
> > > Didn't seem to populate? https://stage.metaphor.io/dataset/8CDF81FE6B8050607E78C53E8B704B05/columns
> > 
> > 
> > I ran it against QA: https://qa.metaphor.io/dataset/8CDF81FE6B8050607E78C53E8B704B05/columns
> 
> I see. Seems like the tags are not populate, probably because the corresponding tags don't exist yet. Consider verify using an existing tag, like `Classification=Public`.

https://stage.metaphor.io/dataset/8CDF81FE6B8050607E78C53E8B704B05/columns
Column tags : Gold, Bronze

File: .github/workflows/configs/custom_governance.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+datasets:
+  - id:
+      platform: SNOWFLAKE
+      name: acme.ride_share.cleaned_bike_rides
+      account: metaphor-dev
+    ownerships:
+      - type: Analytics Owner
+        email: example@domain.com
```
Comment (mars-lan) [inline .github/workflows/configs/custom_governance.yml:8]: Let's use my email address so it'll show up in UI?

File: .github/workflows/configs/custom_governance.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+          - Gold
+          - Bronze
+    descriptions:
+      - description: "This table contains a transformed version of the raw_bike_hires table, which includes additional calculated fields such as creating a duration in minutes field. Each ride has been aggregated so any journey that starts and ends at the same station, in the same month and roughly time of day are aggregated together to get the total minutes similar journeys have taken."
+        email: example@domain.com
+    column_descriptions:
+      - column_name: AVERAGE_DURATION
+        descriptions:
+        - description: "average duration for the bike ride."
+          email: example@domain.com
```
Comment (mars-lan) [inline .github/workflows/configs/custom_governance.yml:23]: ditto

File: .github/workflows/configs/custom_governance.yml (hunk 1 above)
Comment (usefulalgorithm) [reply .github/workflows/configs/custom_governance.yml:8]: sg

Comment (mars-lan) [reply .github/workflows/configs/custom_governance.yml]: Please re-run the action again so the new name shows.

**Comment by usefulalgorithm** [comment]
@mars-lan  let's merge this?

**Review by mars-lan — APPROVED** [review]
Assuming this has been verified end-to-end.

**Pull request merged**